src/utils/daos_hdlr.c

```diff
@@ -1919,6 +1918,49 @@ cont_rollback_hdlr(struct cmd_args_s *ap)
 	return rc;
 }
 
+int
+cont_list_objs_hdlr(struct cmd_args_s *ap)
+{
+	static const int OID_ARR_SIZE	= 8;
+	daos_obj_id_t oids[OID_ARR_SIZE];
+	daos_handle_t oit;
+	daos_anchor_t anchor;
+	uint32_t oids_nr;
+	int rc, rc2 = 0, i;
+
+	/* create a snapshot */
+	rc = cont_create_snap_hdlr(ap);
+	if (rc != 0)
+		goto out;
+
+	/* open OIT */
+	rc = daos_oit_open(ap->cont, ap->epc, &oit, NULL);
+	if (rc != 0) {
+		fprintf(stderr, "open of container's OIT failed: %d\n", rc);
+		goto out_snap;
+	}
+
+	memset(&anchor, 0, sizeof(anchor));
+	while (!daos_anchor_is_eof(&anchor)) {
+		oids_nr = OID_ARR_SIZE;
+		rc = daos_oit_list(oit, oids, &oids_nr, &anchor, NULL);
+		if (rc != 0) {
+			fprintf(stderr, "object IDs enumeration failed: %d\n", rc);
```

Comment:
(style)  line over 80 characters

Reply:
can this be addressed by moving th rc); to next line?  Or the format string?

Reply:
ok